src/Uno.UI/UI/Xaml/Window.Android.cs

```diff
@@ -132,31 +131,9 @@ private double GetLogicalStatusBarHeight()
 
 		private double GetLogicalNavigationBarHeight()
 		{
-			var logicalNavigationBarHeight = 0d;
-			var metrics = new DisplayMetrics();
-			var defaultDisplay = (ContextHelper.Current as Activity)?.WindowManager?.DefaultDisplay;
-
-			var activity = ContextHelper.Current as Activity;
-			var decorView = activity.Window.DecorView;
-
-			var isNavigationBarTranslucent =
-				((int)activity.Window.Attributes.Flags & (int)WindowManagerFlags.TranslucentNavigation) != 0
-				|| ((int)activity.Window.Attributes.Flags & (int)WindowManagerFlags.LayoutNoLimits) != 0;
-
-			if (defaultDisplay != null && IsNavigationBarVisible && isNavigationBarTranslucent)
-			{
-				defaultDisplay.GetMetrics(metrics);
-				var usableHeight = metrics.HeightPixels;
-
-				defaultDisplay.GetRealMetrics(metrics);
-				var realHeight = metrics.HeightPixels;
-
-				logicalNavigationBarHeight = realHeight > usableHeight
-					? ViewHelper.PhysicalToLogicalPixels(realHeight - usableHeight)
-					: 0;
-			}
-
-			return logicalNavigationBarHeight;
+			return NavigationBarHelper.IsNavigationBarVisible && NavigationBarHelper.IsNavigationBarTranslucent
```

Comment:
not "... AND **not** translucent"?

Reply:
So, no. We want to substract it only if it is translucent. The real metrics excluded these measure if they are plain only. Otherwise, it will be like we substract it twice.
I've renamed the method for more comprehension.